var/spack/repos/builtin/packages/argobots/package.py

```diff
@@ -0,0 +1,40 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Argobots(AutotoolsPackage):
+    """Argobots, which was developed as a part of the Argo project, is
+    a lightweight runtime system that supports integrated computation
+    and data movement with massive concurrency. It will directly
+    leverage the lowest-level constructs in the hardware and OS:
+    lightweight notification mechanisms, data movement engines, memory
+    mapping, and data placement strategies. It consists of an
+    execution model and a memory model."""
+    homepage = "http://www.argobots.org/"
+
+    url = "https://github.com/pmodels/argobots/releases/download/v1.0b1/argobots-1.0b1.tar.gz"
```

Comment:
can you add their website 
http://www.argobots.org/

Reply:
It's there on line 36.

Reply:
@healther if there's a recommended whitespace style, I can fixup to follow it. For this package, I just copied an existing `package.py` file and edited it lightly.